### DISCUSSION TITLE
面向人力资源的 GitHub 指南 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#35] [Comments]
# 面向人力资源的 GitHub 指南 | 夜天之书

越来越多的开源软件占领了各个领域依赖链条的关键环节，越来越多的程序员也以参与知名开源软件的开发为荣，将开源贡献和在开源社群当中获得的头衔作为简历当中浓墨重彩的一部分。 在这样的背景下，技术驱动型公司的 HR 也随之需要掌握从开源社群当中找到组织需要的人才的方法。如果企业的业务依托于开源软件的发展，甚至公司直接投入人力参与开源社群，那么 HR 还需要了解开源项目运作的基本流程。 我在最近三年间陆续遇

https://www.tisonkun.org/2022/08/22/github-for-hrs/

**Comment by xxchan** [comment]
关于工具，刚发现 OSS Insight 现在有 reviews 甚至还有 issue_comments 的统计，感觉挺不错
